cdnsystem/storedriver/local/local_driver.go

```diff
@@ -220,22 +189,31 @@ func (ds *diskStorage) Put(ctx context.Context, raw *storedriver.Raw, data io.Re
 		if err = storedriver.CheckTrunc(raw); err != nil {
 			return err
 		}
-		f, err = fileutils.OpenFile(path, os.O_WRONLY|os.O_TRUNC|os.O_CREATE, 0644)
+		if f, err = fileutils.OpenFile(path, os.O_WRONLY|os.O_TRUNC|os.O_CREATE, 0644); err != nil {
+			return err
+		}
 	} else if raw.Append {
-		f, err = fileutils.OpenFile(path, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0644)
+		if f, err = fileutils.OpenFile(path, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0644); err != nil {
+			return err
+		}
 	} else {
-		f, err = fileutils.OpenFile(path, os.O_WRONLY|os.O_CREATE, 0644)
-	}
-	if err != nil {
-		return err
+		if f, err = fileutils.OpenFile(path, os.O_WRONLY|os.O_CREATE, 0644); err != nil {
```

Comment:
Don't use fileutils library, use golang os library。

Reply:
Encapsulate the case that the file does not exist

Reply:
It is not necessary to use this package.